modules/unauthenticated-users-cluster-role-binding.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * authentication/impersonating-system-admin.adoc
+// * authentication/tokens-scoping.adoc
+// * authentication/managing-oauth-access-tokens.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="unauthenticated-users-cluster-role-bindings_{context}"]
+= Adding unauthenticated groups to cluster roles
+
+As a cluster administrator, you can add unauthenticated users to the following cluster roles in {product-title} by creating a cluster role binding. Unauthenticated users do not have access to non-public cluster roles. This should only be done in specific use cases when necessary.
+
+You can add unauthenticated users to the following cluster roles:
+
+* `system:scope-impersonation`
+* `system:webhook`
+* `system:oauth-token-deleter`
+* `self-access-reviewer`
+
+[IMPORTANT]
+====
+Always verify compliance with your organization's security standards when modifying unauthenticated access.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the OpenShift CLI (`oc`).
+
+.Procedure
+
+[Note]
+====
+Replace the `<cluster_role>` variable with the cluster role you want to update.
+====
```

Comment:
This section isn't rendering correctly. Maybe because _Note_ isn't all caps? Or, it doesn't like the admonition after the heading?
![image](https://github.com/openshift/openshift-docs/assets/25209155/0cd40ca8-f3b2-4ae3-8af4-69e5f3a95ce0)